elephant/sta.py

```diff
@@ -314,3 +320,57 @@ def spike_field_coherence(signal, spiketrain, **kwargs):
 
     return (pq.Quantity(sfc, units=pq.dimensionless),
             pq.Quantity(frequencies, units=pq.Hz))
+
+
+def spike_triggered_population_response(spiketrains,
+                                        bin_size=1*pq.ms,
+                                        sigma=5*pq.ms):
+    """
+    Spike triggered population response (stPR) as described in [1]_.
+
+    Parameters
+    ----------
+    spiketrains : list of neo.SpikeTrain
+        list of spike train objects. The spike trains should have the same
+        `t_start` and `t_stop` times.
+    bin_size : pq.Quantity
+        Time quantity for binning of the `spiketrains`.
+        Default: 1*pq.ms
+    sigma : pq.Quantity
+        Standard deviation of gaussian kernel for estimation of population
+        rate.
+        Default: 5*pq.ms
+
+    References
+    ----------
+    .. [1] Okun, M., Steinmetz, N., Cossell, L. et al. Diverse coupling of
+    neurons to populations in sensory cortex. Nature 521, 511–515 (2015).
+    https://doi.org/10.1038/nature14273
```

Comment:
keep an eye on the alignment - the lines should be perfectly aligned with spaces to have the same padding
so it should be
```
    .. [1] Okun, M., Steinmetz, N., Cossell, L. et al. Diverse coupling of
            neurons to populations in sensory cortex. Nature 521, 511–515 (2015).
            https://doi.org/10.1038/nature14273
```